pages/checkout/index.js

```diff
@@ -25,30 +33,84 @@ const useStyles = makeStyles((theme) => ({
   gridItem: {
     marginBottom: theme.spacing(1),
     marginTop: theme.spacing(1),
+    [theme.breakpoints.up("1020")]: {
+      // marginLeft: theme.spacing(22),
+    },
+  },
+  card: {
+    alignItems: 'center',
+    alignContent: 'center',
+    verticalAlign: "middle",
+    backgroundImage: 'url(./_MG_0509.jpg)',
+    backgroundRepeat: 'no-repeat',
+    WebkitBackgroundSize: 'cover',
+    // boxShadow: "0 5px 8px 0 rgba(0, 0, 0, 0.3)",
+    [theme.breakpoints.down("768")]: {
+      marginTop: 75,
+    },
+    [theme.breakpoints.up("768")]: {
+      marginTop: 75,
+      height: '150%',
+    },
+    [theme.breakpoints.up("1020")]: {
+      marginTop: 150,
+      marginLeft: 200,
+      height: '150%',
+    },
+    [theme.breakpoints.up("1025")]: {
+      marginTop: 75,
+      marginLeft: 0,
+    },
+  },
+  second: {
+    background: 'rgba(31,65,69, 0.0)',
+    border: 0,
+    paddingLeft: 20,
+    backgroundColor: 'rgba(255, 255, 255, 0.5)',
+    boxShadow: "0 4px 8px 0 rgba(255, 255, 255, 0.2), 0 6px 20px 0 rgba(255, 255, 255, 0.19);",
+    [theme.breakpoints.up("1020")]: {
+      marginLeft: 0,
+      marginTop: 20,
+    },
+    heading: {
+      fontFamily: 'Oswald Regular',
+    },
   },
 }));
 
-export default function Checkout() {
+export default function Checkout({menuItems}) {
   const { total, cartItems, checkout, handleCheckout } = useContext(CartContext);
 
   const classes = useStyles();
 
   return (
     <Grid container direction="row">
-      <Grid item xs={12} className={classes.gridItem}>
-        <h1>Checkout Page</h1>
-      </Grid>
-      <Grid item xs={12} className={classes.gridItem}>
-        <p>Total: {total}</p>
-      </Grid>
-      <div>
-        <Grid item xs={12} className={classes.gridItem}>
-          <Typography variant="h4" component="h4">
-            Shipping Address
-          </Typography>
-          <Form />
-        </Grid>
-      </div>
+      <TopNavBar></TopNavBar>
+      <Navigation menuItems={menuItems}></Navigation>
+      <Container maxWidth="lg">
+        <Card className={classes.card}>
+          <CardContent>
+            <Card className={classes.second}>
+              <Grid item xs={12} className={classes.gridItem}>
+              <Typography className={classes.heading} variant="h2" component="h1">
+                Checkout
+                </Typography>
+              </Grid>
+              <Grid item xs={12} className={classes.gridItem}>
+                <p>Total: {total}</p>
```

Comment:
This might render as `Total:1234` instead of `Total: 1234` if the space collapses (it happens often).

Try replacing it with 

```
Total:&nbsp;{total}
```